docs/apis-tools/administration-sm-api/administration-sm-api-authentication.md

```diff
@@ -5,34 +5,61 @@ sidebar_position: 2
 description: "The Administration API for Self-Managed is a REST API and provides access to Console Self-Managed data. Requests and responses are in JSON notation."
 ---
 
-To authenticate for the API, generate a JWT token and pass it in each request:
+All Administration Self-Managed API requests require authentication. To authenticate, generate a [JSON Web Token (JWT)](https://jwt.io/introduction/) and pass it in each request.
+
+## Generating a token
 
 1. [Add an M2M application in Identity](/self-managed/identity/user-guide/additional-features/incorporate-applications.md).
 2. [Add permissions to this application](/self-managed/identity/user-guide/additional-features/incorporate-applications.md) for **Administration Self-Managed API**.
-3. [Generate a token](/self-managed/identity/user-guide/authorizations/generating-m2m-tokens.md) to access the REST API. You will need the `client_id` and `client_secret` from the Identity application you created.
+3. Capture the `Client ID` and `Client Secret` from the application in Identity.
+4. [Generate a token](/self-managed/identity/user-guide/authorizations/generating-m2m-tokens.md) to access the REST API. Provide the `client_id` and `client_secret` from the values you previously captured in Identity.
    ```shell
    curl --location --request POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
    --header 'Content-Type: application/x-www-form-urlencoded' \
-   --data-urlencode 'client_id=<client id>' \
-   --data-urlencode 'client_secret=<client_secret>' \
+   --data-urlencode "client_id=${CLIENT_ID}" \
+   --data-urlencode "client_secret=${CLIENT_SECRET}" \
    --data-urlencode 'grant_type=client_credentials'
    ```
-4. You will get something like the following:
+5. A successful authentication response looks like the following:
    ```json
    {
-     "access_token": "eyJhbG...",
+     "access_token": "<TOKEN>",
      "expires_in": 300,
      "refresh_expires_in": 0,
      "token_type": "Bearer",
      "not-before-policy": 0
    }
    ```
+6. Capture the value of the `access_token` property and store it as your token.
 
-## Example usage
+## Using a token
 
-1. Take the **access_token** value from the response object and store it as your token.
-2. Send the token as an authorization header in each request.
+Include the captured token as an authorization header in each request: `Authorization: Bearer <TOKEN>`.
 
-   ```shell
-   curl -o - 'http://localhost:8080/admin-api/clusters' -H 'Authorization: Bearer eyJhb...'
-   ```
+For example, to call the Administration Self-Managed API's ["Get current clusters" endpoint](./specifications/search-tasks.api.mdx), send the following request against the target environment:
+
+:::tip
+The `${CAMUNDA_BASE_URL}` variable below represents the URL of the Self-Managed environment. You can configure this value in your Self-Managed installation. The default value is `http://localhost:8080`.
```

Comment:
@urbanisierung there are a few things here I'm hoping you can clarify for me: 

1. I don't like how I described this: "the URL of the Self-Managed environment." Which component is it associated with? Zeebe?

2. Is 8080 the default port in a self-managed environment? When I spun up an 8.6-alpha4 environment via Docker Compose from the camunda/camunda-platform repo, the port associated with Zeebe was 8088. I'm not sure what other installation methods use as a default. 

3. I usually try to test these auth guides against a real environment, to make sure the scripts work when copy-pasted. With my 8.6-alpha4 environment from Docker Compose, I got a 404 error for the path `/admin-api/clusters`, against any of the running ports on `localhost`. Is there something I need to do to enable this API when setting up my environment?

Reply:
Hey @pepopowitz! Sorry for the late response, I was on FTO.

> I don't like how I described this: "the URL of the Self-Managed environment." Which component is it associated with? Zeebe?

This is all about the admin API of console self-managed. Basically a very first iteration with some basic capabilities as described here: https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/2871

> Is 8080 the default port in a self-managed environment?

Yesyes

> When I spun up an 8.6-alpha4 environment via Docker Compose from the camunda/camunda-platform repo, the port associated with Zeebe was 8088. I'm not sure what other installation methods use as a default.

There is currently no way to use console self-managed with docker compose. The console app relies on information provided by Helm Charts.

> I usually try to test these auth guides against a real environment, to make sure the scripts work when copy-pasted. With my 8.6-alpha4 environment from Docker Compose, I got a 404 error for the path /admin-api/clusters, against any of the running ports on localhost. Is there something I need to do to enable this API when setting up my environment?

The Admin API is enabled for versions starting with `8.6.0-alpha4`. But since there is no docker compose flow, the console pod won't boot up. If you need a quick'n'dirty way to use helm: I've created a very basic (and most probably buggy xD ) way to spawn up a self-managed installation with helm: https://github.com/urbanisierung/csctl.